matching-service/index.js

```diff
@@ -34,6 +43,11 @@ const router = express.Router()
 
 router.post('/', findMatch)
 router.delete('/', deleteMatch)
+router.get('/rooms', findAllRooms)
+router.get('/room', findOneRoom)
+router.post('/room', createRoom)
+router.patch('/room/:room_id', getRoomByRoomId ,updateRoom)

```

Comment:
It doesn't seem like the controller for updating a room makes use of the room fetched by the middleware... could we remove the middleware to also save one extra trip to the db?

Reply:
I don't see it... are you sure HAHA

Reply:
Actually that's a very good question, why do I need to retrieve the room using middleware, when I can directly update the room? (Same situation as the delete below). Also, in the frontend API I don't see it being called, so when will the `GET` and `PATCH` endpoints ever be called?